spec/features/checkouts_spec.rb

```diff
@@ -1,9 +1,23 @@
 require 'rails_helper'
 
 RSpec.feature "Checkouts", type: :feature do
+
+  before(:each) do
+  O1 = Order.create(Status: "Completed", total: 23600,
```

Comment:
Use 2 (not 0) spaces for indentation.<br>Separate every 3 digits in the integer portion of a number with underscores(_).